src/components/option/Options.scss

```
@@ -41,5 +43,15 @@
       width: 44px;
       height: 44px;
     }
+
+    @include mobile {
+      width: 154px;
+      height: 154px;
+    }
+  }
+  @include mobile {
+    //2x2배열을 위함
+    display: grid;
+    grid-template-columns: 1fr 1fr;
```

```suggestion
grid-template-columns: repeat(2, 1fr);
```
똑같지만..! 혹시나 수정될 수도 있는 점 생각해서 repeat이 좋을거같아용

---

바로 수정 완료했습니다~!! 😎👍 감사해용💕